live-examples/js-examples/number-negative-infinity.html

```diff
@@ -0,0 +1,15 @@
+<pre>
+<code id="static-js">function checkNumber(smallNumber) {
+	if (smallNumber === Number.NEGATIVE_INFINITY) { return "Process number as -Infinity" };
+  	return smallNumber;
+}
+
+var x = -Number.MAX_VALUE, x2 = -Number.MAX_VALUE * 2;
```

Comment:
My suggestion here would be to have two explicit `var` declarations. I would say this is a style rule we can agree to but, also for consistency with other code.

```
var x = -Number.MAX_VALUE;
var x2 = -Number.MAX_VALUE * 2;
```

Reply:
OK - yes, just trying to space-save again. But I think we've established that consistency is more important.